source/features/highest-rated-comment.tsx

```diff
@@ -51,7 +51,7 @@ function getBestComment(): HTMLElement | undefined {
 
 function highlightBestComment(bestComment: Element): void {
 	select('.unminimized-comment', bestComment)!.classList.add('rgh-highest-rated-comment');
-	select('.unminimized-comment .timeline-comment-header-text', bestComment)!.before(
+	select('.unminimized-comment .text-normal', bestComment)!.before(
```

Comment:
This selector is too generic now, it could appear anywhere

Reply:
Can fix 🙂
```suggestion
	select('.unminimized-comment .timeline-comment-header h3', bestComment)!.before(
```

Reply:
Resolved by a7c9479...477c01e